firmware/components/driver_sensor_am2320/include/driver_am2320.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Driver for the AM2320 I2C temperature and humidity sensor
+ * Jeroen Bos (Hieronymox)
+ *
+ */
+
+#ifndef DRIVER_AM2320_H
+#define DRIVER_AM2320_H
+
+#include <stdint.h>
+#include <esp_err.h>
+
+#define CACHE_TIMEOUT_MS 10 * (1000 * 1000)  // 10 seconds
+#define SENSOR_NAN_VALUE 0xFFFF
+
+#ifdef CONFIG_DRIVER_AM2320_ENABLE
+	#if CONFIG_I2C_MASTER_FREQ_HZ > 100000
```

Comment:
Please make sure you're not mixing tabs and spaces - the easiest way would be to just format everything using clang-format (there's a `.clang-format` file in the root of the repository)

Reply:
Yeah I must have done that manually, if I run format from my IDE (VSCode) it removes the indent but that seems to be inline with other code so I will commit that.

But regarding the clang-format I did see some codestyle that is not specified in the clang-format. For me to apply the same usage of braces and indent I have to use these extra settings: 

BreakBeforeBraces: Stroustrup
IndentWidth: 4

Is that the correct style? And if so shall I commit these changes to the .clang-format?

Reply:
The missing options should come from the parent style (Chromium). When I run `clang-format --style=file --dump-config` I see `BreakBeforeBraces: Attach` and `IndentWidth: 2`.

What do you mean when you say that you see missing settings? Did something warn you about them, or do you just see that they're not specified explicitly? If something (vscode?) warned you, can you check `clang-format --style=chromium --dump-config` with whatever clang-format binary it uses and check if the two options are in there? With clang-format 10 they're there, at least for me.

Reply:
Ahhh so 2 spaces indent and braces attached are what is intended. :) Yeah that is what I got initially. But I looked at a lot of code and noticed the formatting was not the same as what I got when I applied the formatting. So I assumed some settings where missing in the clang-format file.

Reply:
Yeah, the format file is rather recent, and we decided against formatting the whole repo at once. Instead we do a small formatting commit before touching any new file :)

Reply:
I will reformat my newly added files, but I wont reformat the change in i2c driver because the diff will be much bigger